### ISSUE TITLE
Minutes until departure instead of time

### ISSUE THREAD

**Issue by toco1** [jonkristian/entur-card#24]
It is possible to show the remaining time until departure instead/additionally to the departure time on the right-hand side of the card?

**Comment by jonkristian** [comment]
It's not something that is implemented. However it shouldn't be too hard to do. I will try and find the time to look into this.

**Comment by terjew** [comment]
@jonkristian following up on this, I notice that when there are no delays, I get the text "Arrives in x minutes" below the line number. However, when there is a delay, this text is not shown and instead it shows the number of minutes delay in red on the right. Is there any way to show both?

Example below: 
![image](https://user-images.githubusercontent.com/27724947/208683548-50bbe740-dedb-423d-993c-bc1bc4c4b166.png)

**Comment by jonkristian** [comment]
Yes this makes sense, I will see if I can get it added along with a flag to set remaining time.

By the way, thank you so much for the coffees, highly appreciated!

**Comment by jonkristian** [comment]
> It is possible to show the remaining time until departure instead/additionally to the departure time on the right-hand side of the card?

@toco1 Working on a rewrite and just want to make sure I understood what you were asking, you want an option to replace the time on the right hand side with a countdown like you get via the `show_human` option?

**Comment by jonkristian** [comment]
@toco1 and @terjew this should now be sorted in 2.0.0-beta, sorry for the long wait :smiling_face_with_tear: Keeping this open until it's in stable release, just wanted to let you know if you want to try it out.

**Comment by repvik** [comment]
It doesn't handle midnight well:

<img width="539" alt="image" src="https://github.com/jonkristian/entur-card/assets/1718917/4e41f7e9-b9f3-4769-bcd4-85dba1dc75a7">

**Comment by jonkristian** [comment]
The underlying issue is that entur does not return a datetime, just time string. So I have to get creative here, I will look into how this can be improved.

**Comment by jonkristian** [comment]
@repvik Could you please test the latest from master branch and see how that works? I've rewritten human readable calculations so it should track correctly when time crosses midnight, and I've added a threshold of about 8 minutes for a "departs in..." string too.
